config/sanitizer/sanitizers.cmake

```diff
@@ -83,7 +83,7 @@ if(USE_SANITIZER)
     elseif(MSVC)
       if(USE_SANITIZER MATCHES "([Aa]ddress)")
         message(STATUS "Building with Address sanitizer")
-        append("-fsanitize=address" CMAKE_C_SANITIZER_FLAGS CMAKE_CXX_SANITIZER_FLAGS)
+        append("/fsanitize=address" CMAKE_C_SANITIZER_FLAGS CMAKE_CXX_SANITIZER_FLAGS)
```

Comment:
This should be restricted to VS 2019 16.9

Reply:
VS2019 seems to be the first version to work with "${CMAKE_C_COMPILER_ID}" MATCHES "Clang". There is also a case where a LLVM compiler can be used with changes to VS paths, but it IDs as LLVM.

Reply:
From what I've seen on the interwebs, `/fsanitize=address` was only added in VS 2019 16.9.

https://devblogs.microsoft.com/cppblog/address-sanitizer-for-msvc-now-generally-available/

Reply:
Right, that is what we have on our test machines.

Reply:
Added check for MSVC version.